deployments/ecs/ec2/splunk-otel-collector.json

```diff
@@ -0,0 +1,119 @@
+{
+  "ipcMode": null,
+  "executionRoleArn": null,
+  "containerDefinitions": [
+    {
+      "dnsSearchDomains": null,
+      "environmentFiles": null,
+      "logConfiguration": null,
+      "entryPoint": null,
+      "portMappings": [],
+      "command": null,
+      "linuxParameters": null,
+      "cpu": 0,
+      "environment": [
+        {
+          "name": "ECS_METADATA_EXCLUDED_IMAGES",
+          "value": "[\"quay.io/signalfx/splunk-otel-collector\"]"
+        },
+        {
+          "name": "SPLUNK_ACCESS_TOKEN",
+          "value": "MY_SPLUNK_ACCESS_TOKEN"
+        },
+        {
+          "name": "SPLUNK_CONFIG",
+          "value": "/etc/otel/collector/ecs_ec2_config.yaml"
+        },
+        {
+          "name": "SPLUNK_REALM",
+          "value": "MY_SPLUNK_REALM"
+        }
+      ],
+      "resourceRequirements": null,
+      "ulimits": null,
+      "dnsServers": null,
+      "mountPoints": [
+        {
+          "readOnly": true,
+          "containerPath": "/hostfs",
```

Comment:
Is this acceptable?
```suggestion
          "containerPath": "/",
```

Reply:
@rmfitzpatrick my immediate reaction is no because you effectively overriding the container fs with host fs! I don't see how you then get files specific to the container. @dmitryax what do you think?

Reply:
Right, you are describing potential fallout/the general issue and this is where awareness of the host metrics receiver scraper functionality and the collector's [scratch filesystem](https://github.com/signalfx/splunk-otel-collector/blob/main/cmd/otelcol/Dockerfile#L23) is required.  Using `/hostfs` won't magically resolve the problem unless there's a corresponding config change in the collector, which doesn't appear to exist for anything but the filesystem scraper.

Reply:
@dmitryax it looks like we have to exclude host metrics unless somehow replacing the container fs with host fs is ok which I don't see how that can fly. Or, maybe there is a way to configure the root path in the collector for the scrapers to find the relevant filesystems to use.

Reply:
Host metrics scrapers use this library which is configurable through env vars https://github.com/shirou/gopsutil#usage. Look at k8s configuration as a reference https://github.com/signalfx/splunk-otel-collector-chart/blob/main/helm-charts/splunk-otel-collector/templates/daemonset.yaml#L186-L198

Reply:
Ok, I set the env vars that configure fs paths needed by the scraper.